src/main/java/co/cask/cdap/guides/workflow/PurchaseWorkflow.java

```diff
@@ -0,0 +1,27 @@
+package co.cask.cdap.guides.workflow;
+
+import co.cask.cdap.api.workflow.AbstractWorkflow;
+
+/**
+ * Workflow for processing purchase events and computing total purchases made by particular user and total purchases
```

Comment:
Workflow for processing purchase events and computing total purchases made by a particular user and total purchases